src/commands/incoming-numbers/list.ts

```diff
@@ -11,7 +11,7 @@ export class incomingNumbersList extends Command {
         phoneNumber: flags.string({
             char: "p",
             description:
-                "Only show incoming phone number resources that match this PCRE-compatible  regular expression.",
+                "Only show incoming phone number resources that match this <a href=https://www.pcre.org/ target=blank>PCRE-compatible</a>  regular expression.",
```

Comment:
Override HTML link tag

Reply:
thanks for the catch anything encapsulated in <> are not removed in the generation of the schema